internal/parser/methods.go

```diff
@@ -0,0 +1,57 @@
+package parser
+
+import (
+	"github.com/Joffref/genz/pkg/models"
+	"go/ast"
+	"go/types"
+)
+
+// parseMethods returns a slice of models.Method from the given map of method signatures.
+// It's not responsible for parsing the methods' comments. It should be done by the caller.
+func parseMethods(signatures map[string]*types.Signature) ([]models.Method, error) {
+	methods := make([]models.Method, len(signatures))
```

Comment:
```suggestion
	methods := make([]models.Method, 0, len(signatures))
```
So you can use `append()` and remove `index` variable